source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -6607,6 +6607,7 @@ export interface IMarkdownProcessingResult {
     outputHtml?: any;
 }
 
+// @dynamic
```

Comment:
Can you elaborate what this is, why it is necessary and why only here?

Reply:
in this class there is not a real static field that is not initialized. The static field has only a getter and a setter, so it can't be initialized (at least of my knowledge). 
I found this [issue](https://github.com/ng-packagr/ng-packagr/issues/641) and applied the solution from there

Reply:
So there are a bunch of static events in the AdaptiveCard class, for instance:
```typescript
static onExecuteAction?: (action: Action) => void;
```

Your solution for AdaptiveCard seems to be to decorate it with `// @dynamic` whereas on the `Media` class you opted to initialize the event declaration. Is that correct?

It seems to me that the issue should be solved the same way everywhere. Given that `// @dynamic` seems to be an Angular-specific concept, I think the right solution is to initialize the various static fields in the AdaptiveCard class, and remove the `// @dynamic` directive.

Reply:
I agree that it would be better to use the same approach everywhere. But like I sayed in my comment above, I don't know a solution to rewrite the getter and setter of `processMarkdown `without changing the behavior of the code. The only solution I see is to remove `processMarkdown` (it is deprecated) completely from the class.

Reply:
Oh my mistake, I did read your comment about that, then forgot. Sorry about that. Then I too don't see an alternative for now.